resources.whatwg.org/build/deploy.sh

```diff
@@ -64,6 +63,22 @@ header() {
   echo -e "\x1b[1m$1\x1b[0m"
 }
 
+header "Linting the source:"
+MATCHES=$(
+  grep -niE '( (code|span|var)(>| data-x=)|[^<;]/(code|span|var)>)' "$INPUT_FILE" | perl -lpe 'print "\nPossible copypasta:" if $. == 1'
```

Comment:
data-x is specific to HTML spec, but I guess that's ok.  This check is trying to catch missing the "<" in start or end tag, because the messages from the validator don't point to the cause.